Library/ViewModels/LoginToutViewModel.swift

```diff
@@ -58,6 +79,9 @@ public protocol LoginToutViewModelOutputs {
   /// Emits when a login success notification should be posted.
   var postNotification: Signal<(Notification, Notification), Never> { get }
 
+  /// Emits when Continue with Apple button is tapped.
+  var prepareSignInWithAppleRequest: Signal<Void, Never> { get }
```

Comment:
I noticed that for Facebook login, the corresponding output is called `attemptFacebookLogin`. I wonder if we should try to keep the naming consistent for various 3rd party logins? So maybe `attemptAppleLogin` would be better here?